cmake/setup_cached_variables.cmake

```diff
@@ -177,6 +178,11 @@ option(DEAL_II_UNITY_BUILD
   OFF)
 mark_as_advanced(DEAL_II_UNITY_BUILD)
 
+option(DEAL_II_BUILD_CXX20_MODULE
+  "If set to ON, and if compiler, cmake, and build system are suitable, build a C++20 style module that can be imported instead of using \#include directives."
+  OFF)
```

Comment:
Does that mean that we still build the library from the source files or that we exclusively rely on modules?

Reply:
Yes, sure, that sounds reasonable!